src/users/users.service.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { UsersService } from './users.service';
+import { User } from './user.model';
+import { getModelToken, SequelizeModule } from '@nestjs/sequelize';
+import * as consts from '../../src/test.constants';
+import { Repository } from 'sequelize-typescript';
+
+describe("UsersService Unit Tests", () => {
+    let usersService: UsersService;
+    let module: TestingModule;
+    // Used for the remove() function
+    let userID;
+
+    beforeAll(async () => {
+        module = await Test.createTestingModule({
+            imports: [
+                SequelizeModule.forRoot({
```

Comment:
Need to use the database configuration that is already present in the application.